src/core/linalg/src/sparse/4C_linalg_blocksparsematrix.hpp

```diff
@@ -28,7 +28,7 @@ namespace Core::LinAlg
     implemented in terms of the matrix blocks.
 
    */
-  class BlockSparseMatrixBase : public SparseOperator
+  class BlockSparseMatrixBase : public SparseOperator, public Epetra_Operator
```

Comment:
Why is inheritance from `Epetra_Operator` still required for `BlockSparseMatrixBase`?